sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Vendoring/OpenTelemetry.Instrumentation.AspNetCore/AspNetCoreInstrumentation.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
Who is handling the legal review? Who should bless the relicensing from Apache 2.0 to MIT?

Reply:
I'm following up with CELA on this.

Reply:
The response from CELA:

> For the copied files just leave the 3rd party Apache license header as-is and do not add the Microsoft copyright statement.